www/tutorial/package.json

```diff
@@ -2,12 +2,12 @@
   "name": "angular2-quickstart",
   "version": "1.0.0",
   "scripts": {
-    "start": "tsc && concurrently \"npm run tsc:w\" \"npm run lite\" ",
-    "lite": "lite-server",
-    "postinstall": "typings install",
-    "tsc": "tsc",
+    "build": "webpack",
+    "start": "webpack-dev-server  & open http://localhost:8080/",
+    "dev": "webpack-dev-server --inline --hot & open http://localhost:8080/",
+    "try-dev": "tsc && concurrently \"npm run tsc:w\" \"npm run lite\" ",
```

Comment:
so i guess i should prob take out the extra stuff i put back in for that command. i'll do that a bit later